File: HolaMundoJava/app/src/main/java/com/example/holamundojava/backend/WSProductos.java
package com.example.holamundojava.backend;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WSProductos {
    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("nombre")
    @Expose
    private String nombre;
    @SerializedName("precio")
    @Expose
    private String precio;
    @SerializedName("descripcion")
    @Expose
    private String descripcion;
    @SerializedName("categoria")
    @Expose
    private String categoria;

    public String getId() {
        return id;
    }
    public void setId(String id) {
        this.id = id;
    }
    public String getNombre() {
        return nombre;
    }
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    public String getPrecio() {
        return precio;
    }
    public void setPrecio(String precio) {
        this.precio = precio;
    }
    public String getDescripcion() {
        return descripcion;
    }
    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }
    public String getCategoria() {
        return categoria;
    }
    public void setCategoria(String categoria) {
        this.categoria = categoria;
    }
}

File: HolaMundoKotlin/app/src/main/java/com/example/holamundokotlin/backend/WSProductos.kt
package com.example.holamundokotlin.backend

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WSProductos {
    @SerializedName("id")
    @Expose
    var id: String = ""

    @SerializedName("nombre")
    @Expose
    var nombre: String = ""

    @SerializedName("precio")
    @Expose
    var precio: String = ""

    @SerializedName("descripcion")
    @Expose
    var descripcion: String = ""

    @SerializedName("categoria")
    @Expose
    var categoria: String = ""
}